doc/help/configuring.asciidoc

```diff
@@ -235,7 +235,7 @@ following snippet to `config.py`:
 
 [source,python]
 ----
-config.load_autoconfig()
+config.load_autoconfig(True)
```

Comment:
Do you think the `True` is useful here? While it's a bit more consistent with `config.load_autoconfig(False)`, I think it might be rather confusing, and `config.load_autoconfig()` might be more expressive - but I'm not entirely sure. Opinions welcome!

Reply:
I just thought of adding the `True` for consistency, but I get that it seems confusing, if you want I can change it.

Reply:
I'd say let's remove it then.